cocos/ui/UIEditBox/Mac/CCUIEditBoxMac.mm

```diff
@@ -135,6 +135,8 @@ - (NSWindow *)window
 - (void)openKeyboard
 {
     [self.window.contentView addSubview:self.textInput];
+    //FIXME: it will cause a few OpenGL error once, should be fix when glfw version upgraded.
```

Comment:
Looks good to me. I will merge it, but i think it is better to create an issue in cocos2d-x that revert the codes when GLFW fixes the issue.